packages/core/admin/admin/src/pages/Home/components/ContentManagerWidgets.tsx

```diff
@@ -118,21 +118,49 @@ const LastEditedWidget = () => {
   const { data, isLoading, error } = useGetRecentDocumentsQuery({ action: 'update' });
 
   if (isLoading) {
-    return <Widget.Loading />;
+    return (
+      <Widget.Root
+        title={{
+          id: 'content-manager.widget.last-edited.title',
+          defaultMessage: 'Last edited entries',
+        }}
+        icon={Pencil}
+      >
+        <Widget.Loading />
+      </Widget.Root>
+    );
   }
```

Comment:
I don't really like this approach for 2 reasons:
- this means recreating the root component everytime the state changes, even though that part of the ui is constant. it means some rerenders/repaints that can be avoided
- it's a bit further from the way homepage will work when we add the API. It's Strapi that's gonna render the Root component, not the widget developers, they'll only have access to the content box inside of it.

So I would recommend something closer to what I had on the 1st widget PR:
- a LastEditedContent + LastPublishedContent component that manages the different states (this is the part that will later get registered in the API)
- a LastEditedWidget that just wraps the Content in the Root (this will be the job of the admin in the future)

so I would recommend something

Reply:
>this means recreating the root component everytime the state changes, even though that part of the ui is constant. it means some rerenders/repaints that can be avoided

I see what you mean makes sense.

>a LastEditedContent + LastPublishedContent component that manages the different states (this is the part that will later get registered in the API)

I find it unfortunate to create and maintain two components that are pretty much the exact same thing but if that's how you want to proceed, works for me

Reply:
I'm happy to discuss it!

> pretty much the exact same thing

Are you referring to LastEditedContent + LastPublishedContent being very similar? or the split between the Widget and the Content?

Reply:
LastEditedContent + LastPublishedContent <-- this one

I started working on it and I think your suggestion has us down the right path, I'll implement it that way then maybe we can take a quick look together tomorrow to see if we can refactor anything 👍

Reply:
sounds good!